pkgs/desktops/e19/default.nix

```diff
@@ -1,12 +1,11 @@
 { callPackage, pkgs }:
 let
-  openjpeg_1 = with pkgs; lib.overrideDerivation openjpeg (oldAttrs: {
-         name = "openjpeg-1.5.1";
+  openjpeg_1 = with pkgs; lib.overrideDerivation openjpeg (oldAttrs: rec {
+         name = "openjpeg-1.5.2";
          src = fetchurl {
-           url = "http://openjpeg.googlecode.com/files/openjpeg-1.5.1.tar.gz";
-           sha1 = "1b0b74d1af4c297fd82806a9325bb544caf9bb8b";
+           url = "mirror://sourceforge/openjpeg.mirror/${name}.tar.gz";
+           sha1 = "lahbqvjpsfdxsrm0wsy3pdrp3pzrjvj9";
          };
-         #passthru = { incDir = "openjpeg-1.5.1"; };
      });
```

Comment:
@vcunat Thank you very much for pointing that out! I introduced it because something depended on `openjpeg 1.x`, but only `openjpeg 2.x` was available in `nixpkgs` back then. Now, we should use the `openjpeg 1.x` from `all-packages.nix`, that's right! Maybe we should also check whether enlightenment and efl work with `openjpeg 2.x` already?